castle/cms/profiles/default/browserlayer.xml

```diff
@@ -5,5 +5,5 @@
     />
   <layer
     name="castle.cms.pwexpiry"
-    interface="castle.cms.pwexpiry.interfaces.ICollectivePWExpiryLayer" />
+    interface="castle.cms.interfaces.passwordvalidation.ICollectivePWExpiryLayer" />
```

Comment:
Reference previous comment, can we remove?

Reply:
Removed.